### PR TITLE
Fix LSP dependency

### PR DESCRIPTION
**Pull request by jgellin-sf**
### What does this PR do?
VSCode has made some dodgy version changes with their latest updates that have broken our build. This PR pins us to a specific version of a dependency to work around the problem.

### What issues does this PR fix or reference?
@W-8582993@

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,6 +22,9 @@
     "tslint": "^5.8.0",
     "typescript": "^4.0.3"
   },
+  "resolutions": {
+    "**/vscode-languageserver-protocol": "3.15.3"
+  },
```
Comment (jgellin-sf) [inline package.json:27]: Yarn will not automatically hoist the pinned version of a dependency of a dependency, even when it is in the accepted range for resolution. This field overrides yarn's default behavior.

**Comment by dobladez** [comment]
LGTM

**Review by dehru — APPROVED** [review]

**Comment by dehru** [comment]
Do we need to publish this?

**Comment by jgellin-sf** [comment]
Probably, but there's no version bump in this PR. My intention was to unblock development, and this is a start.

**Pull request merged**